try:
    number1 = 10
    number2 = 0

    result = number1/number2
    print(result)
except Exception as e:
    print("Exception Arise:", e)

print("Hello, world!")